eq-author/src/App/page/Design/QuestionPageEditor/AnswersEditor/AnswerEditor/index.js

```diff
@@ -116,13 +116,7 @@ class AnswerEditor extends React.Component {
 
   getAnswerTypeText(answer) {
     if (answer.type === UNIT) {
-      const unitConfig = unitConversion[answer.properties.unit];
-      return (
-        <>
-          {unitConfig.unit}
-          {` (${unitConfig.abbreviation})`}
-        </>
-      );
+      return <></>;
```

Comment:
Need to replace the title of the question with something now that we have removed centimetres as the default. 

The import above might need to be removed as well

Reply:
I've removed the check for UNIT so default title is used